CityOfDetroit/bloom
# 1422/finder scaffolding + Q1

# Pull Request Template

## Issue Overview

This PR primarily addresses #1422 and #1433. Technically, #1423 as well which is handled by creating the feature branch.


## Description

This PR implements the path to the "/finder" page with a new button on the homepage, handles the design of the listing finder with the bordered field, progressNav and stepHeader which were brought over from Core, and the functionality of filtering the listings based on user's form choices for Q1. It also establishes a pattern how the form data will be stored across sections of the form and how to limit redundant code. This is subject to potentially change as more questions are implemented but it serves as an initial approach to how the user inputs could be handled.

Two thing to verify on Design @slowbot:
1) Button sizes seemed a bit bigger in the Figma than the ones I implemented, but Detroit's is-big button looks like a box (overrides might be missing the is-big case). Let me know if you want me to pursue the bigger buttons or if the current implementation looks good to you.

## How Can This Be Tested/Reviewed?

This can be tested by going to the detroit homepage, selecting the new 'Find listings for you' button, and doing one of two things: 1) clicking skip and seeing the All rentals view 2) filling out the form and clicking submit to see your selections reflected in the filtered listings page. 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
